flutter_app/test/command_constructor_test.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:dart_mavlink/mavlink.dart';
+import 'package:dart_mavlink/dialects/common.dart';
+import 'package:flutter_app/command_constructor.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Command Constructor Tests', () {
+    var sequence = 0;
+    var systemID = 1;
+    var componentID = 0;
+    var messageID = 33; // GLOBAL_POSITION_INT
+    var interval = 1000000; // Time interval in microseconds == 1 second
+    var requestMessageCommandNumber = 512; // MAV_CMD (MavLink Command) Number for requesting a single instance of a particular MAVLink message ID
+    var setIntervalCommandNumber = 511; // MAV_CMD (MavLink Command) Number for setting an interval between messages for a particular MAVLink message ID
+    var dialect = MavlinkDialectCommon();
```

Comment:
Make this `const` and move it to the top.

Reply:
I don' think I can. "The constructor being called isn't a const constructor."